contingency/contingency-api/src/main/java/com/powsybl/contingency/contingency/list/AbstractLineCriterionContingencyList.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.contingency.contingency.list;
+
+import com.powsybl.contingency.contingency.list.criterion.PropertyCriterion;
+import com.powsybl.contingency.contingency.list.criterion.RegexCriterion;
+import com.powsybl.contingency.contingency.list.criterion.TwoCountriesCriterion;
+import com.powsybl.contingency.contingency.list.criterion.TwoNominalVoltageCriterion;
+import com.powsybl.iidm.network.IdentifiableType;
+
+import java.util.List;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public abstract class AbstractLineCriterionContingencyList extends AbstractEquipmentCriterionContingencyList {
+    private final TwoCountriesCriterion twoCountriesCriterion;
+    private final TwoNominalVoltageCriterion twoNominalVoltageCriterion;
+
+    protected AbstractLineCriterionContingencyList(String name, IdentifiableType type, TwoCountriesCriterion twoCountriesCriterion,
+                                            TwoNominalVoltageCriterion twoNominalVoltageCriterion,
+                                            List<PropertyCriterion> propertyCriteria, RegexCriterion regexCriterion) {
+        super(name, type, propertyCriteria, regexCriterion);
+        this.twoCountriesCriterion = twoCountriesCriterion;
+        this.twoNominalVoltageCriterion = twoNominalVoltageCriterion;
```

Comment:
Should be not null?

Reply:
I think it can be null because if we want the lines between two countries whatever their voltage level is, it should be null, is it ok ?